frontend/src/reducers/user.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const user = createSlice({
+    name: 'user',
+    initialState: {
+        userID: null,
+        accessToken: null,
+        errors: null
+    },
+        reducers: {
+            setUser: (store, action) => {
+                const { userID, accessToken, errors } = action.payload;
+                store.userID = userID
+                store.accessToken = accessToken
+                store.errors = errors
+            }
+        }
+})
+
+export default user;
```

Comment:
You need to store your user in the `localStorage` to prevent logging out after refreshing.